tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -562,6 +562,14 @@ clean:
 clean_downloads:
 	rm -rf $(MAKEFILE_DIR)/downloads
 
+# Print internal variables (used by CMSIS-Pack build)
+printvar-%: 
+	@echo '$*=$($*)'
+
+# Print variable with enclosing tags (used by CMSIS-Pack build)
+printvar-tagged-%: 
```

Comment:
can we keep the <LIST> tags external to the Makefile? I'd like to have the rules in the makefile be as generic as possible.